tools/dialignr/.shed.yml

```diff
@@ -0,0 +1,11 @@
+name: dialignr
+owner: galaxyp
+remote_repository_url: "https://github.com/galaxyproject/tools-iuc/tree/master/tools/dialignr"
+homepage_url: "https://github.com/shubham1637/DIAlignR"
+long_description: |
+ DIAlignR is an R package for retention time alignment of targeted mass spectrometric data, including DIA and SWATH-MS data. This tool works with MS2 chromatograms directly and uses dynamic programming for alignment of raw chromatographic traces. DIAlignR uses a hybrid approach of global (feature-based) and local (raw data-based) alignment to establish correspondence between peaks.
+categories:
+- "DIA Analysis"
```

Comment:
You need to choose an existing category: `Proteomics`?

Reply:
Thanks!!